tests/Unit/DataTableTest.php

```diff
@@ -91,13 +94,20 @@ public function testDataTableState(): void
         $state->setStart(5);
         $state->setLength(10);
         $state->setGlobalSearch('foo');
-        $state->setOrderBy([[0, 'asc'], [1, 'desc']]);
+        $state->setOrderBy([
```

Comment:
As noted at https://github.com/omines/datatables-bundle/issues/388#issuecomment-3104640637 I would revert injecting semi-SQL here as it's confusing. Do the input sanitation tests in the new test case.